### DISCUSSION TITLE
Change Collection Name from Capital letter to Small letter

### DISCUSSION THREAD

**Discussion by suhaimim** [directus/directus#9938] [Archive]
Is it possible to change/modify the collection name once its already created?

![image](https://user-images.githubusercontent.com/35832684/142721440-f1ace72f-179d-4953-a8a4-821620f83775.png)

**Comment by benhaynes** [comment]
Not the key, no. But you can rename the app label using the collection translations option.

**Reply by suhaimim** [reply]
OK thank you. 
However, if I manually modify the column name in the SQL, is that bring any issues? Like API will not working etc.?
Actually, when I am trying to make FETCH the API, I did a mistakes by calling the collection name by Small Letter as my usual practice.
In this case, I miss type the collection name using the Small letter while it was actually a Capital letter.
That is why I am plan to standardize the collection name to be all Small letters which I think it is also the column name to avoid miss during API call.

**Reply by benhaynes** [reply]
You can change things in the database directly, but that will unlink it from any configuration within Directus. Directus uses the database table name as the key for assigning settings and properties, so changing it will remove that connection.

**Reply by rijkvanzanten** [reply]
You could theoretically change it directly in the database. Be sure to check and update the various directus_* collections that might contain any reference to the original name.

**Reply by suhaimim** [reply]
If that is the only way, than its fine. I ll proceed with further test and see what is the best practice. Thank you.
